pkg/injector/patcher/sidecar_container_test.go

```diff
@@ -30,6 +30,8 @@ import (
 	"github.com/dapr/kit/ptr"
 )
 
+const daprImage = "daprio/dapr"
```

Comment:
Nit: refrain from using constants in tests as they can potentially hide data values long term, and is always best to be literal in expression assertions.

```suggestion
```

Reply:
These were due to static check complaints. Should I just suppress them?

Reply:
Yes please. Best is to put the nolint tag above `package` at the top.